source-azure-blob-storage/.gitignore

```diff
@@ -0,0 +1 @@
+.env
```

Comment:
I'm using dot env files for testing, see `source-azure-blob-storage/main_test.go`

Reply:
Ideally we would use a local emulator to run these test, so that they can run automatically in CI and not incur any charges that would be associated with a more permanent "real" azure setup.

Could we [Azurite](https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azurite?tabs=docker-hub%2Cblob-storage) as a local emulator for tests? We use localstack like this for `source-s3`, and have an [advanced endpoint config option](https://github.com/estuary/connectors/blob/a54358b5ef852d5103a421e7ad30da3ef3e7190e/source-s3/main.go#L38) for it. The tests get run [here](https://github.com/estuary/connectors/tree/main/tests/source-s3).

Reply:
Yes I can. I didn't know that Azure has an emulator, thanks for letting me know.